beta-src/src/components/map/WDMapController.tsx

```diff
@@ -84,28 +78,16 @@ const WDMapController: React.FC = function (): React.ReactElement {
         .on("wheel", (e) => e.preventDefault())
         .call(d3Zoom)
         .call(d3Zoom.transform, d3.zoomIdentity.translate(x, y).scale(scale))
-        .on("dblclick.zoom", null)
-        .on("click", (e) => {
-          handleClick[0](e);
-        })
-        .on("dblclick", (e) => {
-          handleClick[1]();
-          handleClick[0](e);
-        });
+        .on("dblclick.zoom", null);
     }
   }, [svgElement, viewport]);
 
-  const apiStatus = useAppSelector(gameApiStatus);
-  let data: GameDataResponse["data"];
-  let members: GameOverviewResponse["members"];
-  if (apiStatus === "succeeded") {
-    ({ data } = useAppSelector(gameData));
-    ({ members } = useAppSelector(gameOverview));
-  }
-
   React.useLayoutEffect(() => {
-    drawUnitsOnMap(members, data);
-  }, [svgElement]);
+    if (data && members) {
+      drawUnitsOnMap(members, data);
+      dispatch(gameApiSliceActions.highlightMapTerritories());
+    }
+  }, [svgElement, data]);
```

Comment:
do we need svgElement here if its not being used in this function? Also don't we need members in here? [members, data]

Reply:
> do we need svgElement here if its not being used in this function? Also don't we need members in here? [members, data]

Nope, I think we originally needed it here but not anymore, and you're right we should add members here. Will make that update.